README.md

```diff
@@ -1,3 +1,9 @@
 # NorthstarDiscordRPC
 
 Plugin to integrate Northstar with Discord's SDK to enable [rich presence](https://discord.com/rich-presence).
+
+# Building
+
+The Northstar RPC plugin uses [cargo](https://doc.rust-lang.org/cargo/index.html) to download dependencies and build code
+
+Run `cargo build --release` in the root directory of the plugin. The output will be found in `target/release`.
```

Comment:
release is not required to build the plugin, it only disables the plugin from throwing errors and forces it to shutdown instead and ofc it optimizes it more.

Reply:
I though @itscynxx had issues with getting the plugin running without compiling as release? Or did I confuse something here? ^^"

Reply:
`cargo build` should work fine. idk why someone would have issues unless they are building on linux :|

Reply:
So running `cargo build` worked just fine for me on Windows.